packages/libkcapi/af_alg-test/runtest.sh

```diff
@@ -29,8 +29,6 @@ rlJournalStart
         rlRun "git clone '$GIT_URL' libkcapi"
         rlRun "(cd libkcapi && git checkout $GIT_REF)"
         rlRun "(cd libkcapi && autoreconf -i)"
```

Comment:
The `autoreconf` line is still needed?

Reply:
Yes, the `test-invocation.sh` script needs the `configure` script to be generated already.